source/val/validate_misc.cpp

```diff
@@ -42,10 +42,10 @@ spv_result_t ValidateUndef(ValidationState_t& _, const Instruction* inst) {
 
 spv_result_t ValidateShaderClock(ValidationState_t& _,
                                  const Instruction* inst) {
-  const uint32_t execution_scope = inst->word(3);
-  if (auto error = ValidateExecutionScope(_, inst, execution_scope)) {
-    return error;
-  }
+  // const uint32_t execution_scope = inst->word(3);
```

Comment:
I'd prefer #if 0..#endif
But c'est la vie

Reply:
I'll change it, no problem.